torchtext/experimental/vectors.py

```diff
@@ -1,7 +1,519 @@
 import csv
+import logging
+import os
+
 import torch
 from torch import Tensor
 import torch.nn as nn
+from tqdm import tqdm
+
+from torchtext.utils import (
+    download_from_url,
+    extract_archive
+)
+
+logger = logging.getLogger(__name__)
+
+CHECKSUMS_GLOVE = {
```

Comment:
I would move those two checksums to the end of the file.

Reply:
Sure